package.json

```diff
@@ -8,13 +8,14 @@
     "@turf/line-intersect": "^6.0.2",
     "@turf/points-within-polygon": "^5.1.5",
     "@types/node": "^13.11.1",
+    "jest": "^26.6.3",
     "lerna": "^3.20.2"
   },
   "scripts": {
     "create_example": "node packages/geoprocessing/dist/scripts/init/createExampleProject.js"
   },
   "dependencies": {
-    "@seasketch/geoprocessing": "^0.8.4",
+    "@seasketch/geoprocessing": "^0.9.0",
```

Comment:
It seems weird that this is a dependency here in the actual geoprocessing project.  I only noticed it because by default it seems that example-project in running unit tests will use this version of GP, which is 0.8.4 currently, and for me produced an obscure babel error by webpack.  bumping this in the root to 0.9 resolves it.  So naively it seems this should get bumped when the gp library gets bumped, but it begs the question should this be here?  Is there a code smell.  cc @underbluewaters

Reply:
0.8.4 is actually a few versions old

Reply:
I also noticed that example-project has a dependency of
`"@seasketch/geoprocessing": "^0.9.0",`

And after running `npm install` in example-project it has its own version of geoprocessing in its node_modules.  Typically in a monorepo setup like this you'll see the tool, like lerna, automatically symlink over to the peer geoprocessing, which it seems like what we really want.  Otherwise I can see in past publish commits that this example-project version of gp is getting manually bumped.
Typically in a monorepo setup I'll

Reply:
Actually maybe the peer bump is happening automatically by lerna with publish command and we don't need the *

Reply:
Looks like it got added by you here Chad in this commit and can probably just go away?  https://github.com/seasketch/geoprocessing/blob/0bff438d16912e1c11fa439ab5057ed73af25f75/package.json#L17

Reply:
So I think what's happening is that without installing geoprocessing as a dependency, then running `npm test` in example-project, it will import the sibling geoprocessing package, which then does not have a `dist`folder with scripts, and it errors out.  Worth making sure I'm understanding and this can be duplicated.

```
 ~/s/geoprocessing/p/example-project  bug/twelch/missing-jest +2 !3  npm test                                                         ✔ 

> example-project-new@0.9.0 test
> geoprocessing test

chmod: /Users/twelch/src/geoprocessing/packages/geoprocessing/dist/scripts/geoprocessing.js: No such file or directory
npm ERR! code 1
npm ERR! path /Users/twelch/src/geoprocessing/packages/example-project
npm ERR! command failed
npm ERR! command sh -c geoprocessing test

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/twelch/.npm/_logs/2021-02-22T21_56_26_174Z-debug.log

 ~/s/geoprocessing/p/example-project  bug/twelch/missing-jest +2 !3  
```

The code smell to me is that the geoprocessing command expects that the `dist` folder exists, and that there are scripts there to run.  I'm not sure what the right behavior should be, perhaps it should optionally not look in dist, and know when it's running in the monorepo.

Reply:
Dan helped me out here, tsc needs to be run (whether watch scripts or not) to generate the dist.  This still feels a little weird but it's working now

Reply:
I just removed it from root altogether

Reply:
Looks like `prepare` is the ticket, `prepublish` is deprecated.  Thanks for the suggestion